lib/Listeners/LoadAdditionalScriptsListener.php

```diff
@@ -25,9 +25,14 @@
 
 namespace OCA\GroupFolders\Listeners;
 
+use OCA\Files\Event\LoadAdditionalScriptsEvent;
+use OCA\Files_Sharing\Event\BeforeTemplateRenderedEvent;
 use OCP\EventDispatcher\Event;
 use OCP\EventDispatcher\IEventListener;
 
+/**
+ * @template-implements IEventListener<LoadAdditionalScriptsEvent|BeforeTemplateRenderedEvent>
```

Comment:
It never works when I use `|` is that situation. Any trick or it simply worked for you?